### PR TITLE
Homework1: Vladimir Rovba

### PR DESCRIPTION
**Pull request by vm128**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle).
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks.
- [ ] my PR made from the fork feature branch to appropriate branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/HomeWork1/hw1_task10.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from time import localtime
+
```
Comment (belovmd) [inline src/HomeWork1/hw1_task10.py:2]: 1 line between import and code

File: src/HomeWork1/hw1_task13.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import unittest
```
Comment (belovmd) [inline src/HomeWork1/hw1_task13.py:1]: 2 lines between import and code

File: src/HomeWork1/hw1_task11.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+REFRAIN = '''
+%d bottles of beer on the wall,
+%d bottles of beer,
+take one down, pass it around,
+%d bottles of beer on the wall!
+'''
+bottles_of_beer = 9
```
Comment (belovmd) [inline src/HomeWork1/hw1_task11.py:8]: we should have at least 1 line between constant and code

**Review by belovmd — CHANGES_REQUESTED** [review]
failed sanity checks - missed docstrings

File: src/HomeWork1/hw1_task13.py (hunk 2 above)
Comment (vm128) [reply src/HomeWork1/hw1_task13.py:1]: Здесь вроде две строки между импортом и кодом. И должно же ведь быть две?

Comment (belovmd) [reply src/HomeWork1/hw1_task13.py:1]: it should be similar

**Review by belovmd — CHANGES_REQUESTED** [review]
missed tasks description
redundant code in pr - hw2

**Review by belovmd — COMMENTED** [review]
not addressed review - no docstrings, file from hw2

**Pull request closed without merging**